### DISCUSSION TITLE
How can i add my sveltekit project ?

### DISCUSSION THREAD

**Discussion by Aliwahid17** [svelte-society/sveltesociety.dev#363] [Q&A]
I recently created a sveltekit project and just want to how I can include that in your list.

Github repo :- [https://www.github.com/aliwahid17/portfolio](https://www.github.com/aliwahid17/portfolio)
Website :- [https://www.wahidali.dev](https://www.wahidali.dev)

**Comment by stephane-vanraes** [comment]
The svelte society website hosts links to templates, components and tools as well as events and recipes.  It does not showcase individual websites.

**Reply by Aliwahid17** [reply]
It is a template which anyone uses with little changes. 

I use it as my personal blog website .
 
My repo is like joy of code repo. Who use it as it blogging website and made it open so other can use also.

**Reply by stephane-vanraes** [reply]
If you want it to be listed as a template it is best to fork it into a separate repo and remove any code or text that is related to you personally and replace with placeholder text such that it is easier to start with.

That repo you can then add to the list in `src/routes/templates/templates.json` by simply making a pull request with that change.

**Reply by Aliwahid17** [reply]
Ok thanks for helping
